docs/using/administering/debugging.rst

```diff
@@ -14,6 +14,35 @@ Ubuntu
 ^^^^^^
 
 XXX This should be documented, see :issue:`1877`.
+# FLOC-2647 Perhaps I should just close 1877 with this branch.
+
+Ubuntu Bug Reporting
+^^^^^^^^^^^^^^^^^^^^
+
+When reporting issues with Flocker on Ubuntu always include copies of all the log files.
+
+The relevant logs can be exported by running the following commands as root:
+
+# FLOC-2647 Perhaps this would be better as a standalone script.
+# Followup issue or in this branch?
```

Comment:
Given the complexity and multiple possibilities for failures, I think it would be better as a script.